src/Components/Title.tsx

```diff
@@ -22,22 +23,16 @@ const Title: React.SFC<TitleProps> = props => {
   const newProps: TitleProps = { ...props }
   delete newProps.titleSize
 
-  return (
-    <div {...newProps}>
-      {props.children}
-    </div>
-  )
+  return <div {...newProps}>{props.children}</div>
 }
 
 const StyledTitle = styled(Title)`
   font-size: ${props => titleSizes[props.titleSize]};
   color: ${props => props.color};
   margin: 20px 0;
-  ${fonts.secondary.style}
-
-  ${media.sm`
+  ${fonts.secondary.style} ${media.sm`
```

Comment:
I wonder if this is prettier that auto-formats code, but at the same time, I'm not sure what the ideal formatting here is.

cc @orta

Reply:
This is prettier doing it, I'd call it as not perfect, but reasonable

Reply:
Ah in the past I've used `;` to separate lines in these instances. Just updated the PR to reflect that!